<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VendorCodeDialog</class>
 <widget class="QDialog" name="VendorCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vendor Code</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color:tomato; font-size:32px; background-color:lightgray;font-weight:600</string>
     </property>
     <property name="text">
      <string>VENDOR CODE</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color:dodgerblue; 
font-size:16px;
 background-color:lightgray;
font-weight:600;</string>
         </property>
         <property name="text">
          <string>VENDOR CODE</string>
         </property>
         <property name="buddy">
          <cstring>vendorcode_spinBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="vendorcode_spinBox">
         <property name="styleSheet">
          <string notr="true">font-size:16px;color:green;</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color:dodgerblue; 
font-size:16px;
 background-color:lightgray;
font-weight:600;</string>
         </property>
         <property name="text">
          <string>NAME</string>
         </property>
         <property name="buddy">
          <cstring>vendorname_lineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendorname_lineEdit">
         <property name="styleSheet">
          <string notr="true">font-size:16px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color:dodgerblue; 
font-size:16px;
 background-color:lightgray;
font-weight:600;</string>
         </property>
         <property name="text">
          <string>ADDRESS</string>
         </property>
         <property name="buddy">
          <cstring>vendoraddress_lineEdit_2</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendoraddress_lineEdit_2">
         <property name="styleSheet">
          <string notr="true">font-size:16px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="add_pushButton">
       <property name="styleSheet">
        <string notr="true">font-size:16px;font-weight:502;</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
